src/tests/ftest/pool/list_verbose.py

```diff
@@ -0,0 +1,332 @@
+        self.assertEqual(len(actual), 1)
+
+        # 10. Destroy pool 1.
+        self.pool[0].destroy()
+
+        # 11. Verify that the list is empty.
+        actual = self.pool[0].dmg.get_pool_list_all(verbose=True)
+        self.assertEqual(len(actual), 0)
+
+    def verify_used_imbalance(self, storage):
+        """Verification steps for test_used_imbalance.
+
+        Args:
+            storage (str): NVME or SCM.
+
+        Returns:
+            list: Errors.
+
+        """
+        # 1. Create a pool with 80GB.
+        if storage == "NVME":
+            self.add_pool(namespace="/run/pool_both/*")
+        else:
+            self.add_pool(namespace="/run/pool_scm_only/*")
+
+        # 2. Store free.
+        before = self.pool.dmg.get_pool_list_all(verbose=True)[0]
+        free_before = -1
+        for usage in before["usage"]:
+            if usage["tier_name"] == storage:
+                free_before = usage["free"]
+
+        # 3, 4. Create an S1 container and run 1GB IOR with S1.
+        cmd_result = self.run_ior_with_pool()
+        metrics = IorCommand.get_ior_metrics(cmd_result)
+        ior_write_size = int(metrics[0][22])
+        self.log.info("IOR metrics = %s", metrics)
+        self.log.info("ior_write_size = %d", ior_write_size)
+
+        # Obtain the new free and imblanace.
+        after = self.pool.dmg.get_pool_list_all(verbose=True)[0]
+        free_after = 0
+        imbalance = 0
+        for usage in after["usage"]:
+            if usage["tier_name"] == storage:
+                free_after = usage["free"]
+                imbalance = usage["imbalance"]
+                break
+
+        # Copy free_before and imbalance = 0 to after so that we can
+        # directly compare.
+        scm_free_before, nvme_free_before = self.get_scm_nvme_free(before)
+        for usage in after["usage"]:
+            if usage["tier_name"] == "NVME":
+                usage["free"] = nvme_free_before
+                usage["imbalance"] = 0
+            elif usage["tier_name"] == "SCM":
+                usage["free"] = scm_free_before
+                usage["imbalance"] = 0
+
+        # 5. Compare before and after except free and imbalance.
+        self.assertDictEqual(before, after)
+
+        # 6, 7. Verify the free and imbalance.
+        errors = []
+        if free_after > free_before - ior_write_size:
+            errors.append(
+                "{} free is bigger than expected! Before {}; After {}".format(
+                    storage, free_before, free_after))
+
+        # Imbalance can be 10 or 11 (expecting 10% for S1 with 10GB/target pool)
+        if imbalance not in [10, 11]:
+            errors.append(
+                "Unexpected {} imbalance! Expected 10 or 11; Actual {}".format(
+                    storage, imbalance))
+
+        # 8. Destroy the container and the pool.
+        self.destroy_containers(containers=self.container)
+        self.destroy_pools(pools=self.pool)
+
```

Comment:
You'll want to reset the `self.pool` and `self.container` lists either here or at the beginning of this method.